db/schema.rb

```diff
@@ -34,12 +34,12 @@
   add_index "tweets", ["user_id", "created_at"], name: "index_tweets_on_user_id_and_created_at"
 
   create_table "users", force: :cascade do |t|
-    t.string   "name",            limit: 255
-    t.string   "email",           limit: 255
-    t.string   "password_digest", limit: 255
+    t.string   "name"
+    t.string   "email"
+    t.string   "password_digest"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "remember_token",  limit: 255
+    t.string   "remember_token"
```

Comment:
__[rubocop]__
* Put one space between the method name and the first argument.
* Prefer single-quoted strings when you don't need string interpolation or special symbols.